src/coaster_gui.cpp

```diff
@@ -626,23 +626,31 @@ void CoasterBuildWindow::SetupSelection()
 	this->piece_selector.pos_piece.piece = nullptr;
 }
 
-void CoasterBuildWindow::SelectorMouseMoveEvent(Viewport *vp, const Point16 &pos)
+/**
+ * Handles coaster build mouse movement events
+ * @param vp Viewport
+ * @param pos Mouse position
+ * @return True if the event no longer needs to be handled (currently always false)
+ */
+bool CoasterBuildWindow::SelectorMouseMoveEvent(Viewport *vp, const Point16 &pos)
```

Comment:
For the Path Gui override, return behavior is actually different. The others you may be correct about.